pandas/io/sql.py

```diff
@@ -834,7 +834,11 @@ def _create_table_setup(self):
                    for name, typ, is_index in column_names_and_types]
 
         if self.keys is not None:
-            pkc = PrimaryKeyConstraint(self.keys, name=self.name + '_pk')
+            if isinstance(self.keys, string_types):
```

Comment:
maybe you want to ensure list-like first ``com.is_list_like``, then stringify? (then you can join by ',' if necessary)